flink-runtime-web/src/main/java/org/apache/flink/runtime/webmonitor/history/HistoryServerArchiveFetcher.java

```diff
@@ -69,6 +69,37 @@
  */
 class HistoryServerArchiveFetcher {
 
+	/**
+	 * Possible job archive operations in history-server.
+	 */
+	public enum JobArchiveOperation {
+		/** Job archive was found in one refresh location and created in history server. */
+		CREATED,
+		/** Job archive was deleted from one of refresh locations and deleted from history server.*/
+		DELETED
+	}
+
+	/**
+	 * Representation of job archive event.
+	 */
+	public static class JobArchiveEvent {
```

Comment:
We're only dealing with jobs in this component, so `ArchiveEvent` would be more succinct.

Reply:
Renamed. Thank you